_config.yml

```diff
@@ -36,6 +36,12 @@ twitter:
 # Build settings
 markdown: kramdown
 theme: minima
+kramdown:
+  parse_block_html: false
```

Comment:
Turned this to `true` means that markdown parsing is enabled within `<details>` blocks. However it is not working well. The closing HTML tags are not parsed.

This introduces problems with other EIPs which use the `details` block properly (820, 1820, etc.): only with a larger code snippet and not with embedded markdown.

The resolution is to change EIP-1474 instead, which was the culprit triggering this entire PR.

Reply:
I can update EIP-1474 accordingly.

Reply:
I already did in this PR.